src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs

```diff
@@ -13,6 +13,7 @@
 using System.Threading;
 using Pretzel.Logic.Extensibility;
 using Xunit;
+using System.ComponentModel.Composition;
```

Comment:
It should be `System.Composition`.

Reply:
There is no need to export the test classes at all. Fixed.